etc/openqa/openqa.ini

```diff
@@ -88,6 +88,20 @@
 # disable auditing of chatty events by default
 blacklist = job_grab job_done
 
+# Sets the storage duration in days for the different audit event types
+[audit/storage_duration]
+# By default cleanup is disabled, see http://open.qa/docs/#_auditing_tracking_openqa_changes
+# The following categories with example values can be uncommented as needed
+#startup = 10
+#jobgroup = 365
+#jobtemplate = 365
+#table = 365
+#iso = 60
+#user = 60
+#asset = 30
+#needle = 30
+#other = 15
```

Comment:
WDYT about being explicit and adding a suffix "_days"?

Reply:
The comment above already states that the durations are measured in days. I don't like a redundant suffix for each of those keys.